public/js/stores/graph.test.js

```diff
@@ -0,0 +1,26 @@
+import { audioGraphNode } from '../graph';
+import { audioNodeNames } from '../graph/node';
+import { GRAPH } from '../actions';
+import graphStore from './graph';
+
+describe('graphStore', () => {
+  test('given no audio nodes, should initalize to an empty array', () => {
+    expect(graphStore.getState()).toEqual([]);
+  });
+
+  test('given an empty store, when a CONNECT action is dispatched, should contain one audio node', () => {
+    // arrange
+    const gainNode = audioGraphNode({ type: audioNodeNames.gain }, 0);
+    const connect = {
+      type: GRAPH.CONNECT,
+      node: gainNode,
+    };
+
+    // act
+    graphStore.dispatch(connect);
+
+    // assert
+    expect(graphStore.getState()).toHaveLength(1);
+    expect(graphStore.getState()).toEqual([gainNode]);
+  });
```

Comment:
@cchester11  I would like you to add more tests to this file. Think of valuable ways we can test this ```graph store```.  We should be able to tests the ability to ```connect``` and ```disconnect```.  Also node order within the graph is very important as well.